samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -96,9 +89,13 @@ static const char location_get_usage_str[] =
 	"                              Zero means timeout is disabled.\n"
 	"  --cellular_service, [str]   Used cellular positioning service:\n"
 	"                              'any' (default), 'nrf' or 'here'\n"
+	"  --cellular_resp_disabled,   Don't wait for cellular positioning response from cloud.\n"
```

Comment:
I would put these out from the default, i.e. flag with CONFIG_LOCATION_SERVICE_EXTERNAL.

Reply:
I guess I could do that. I'm a bit hesitant because I'm not sure we've done this before for command line options...
So do we want to flag it, or should I return error if it's used when external service is not configured?

Reply:
I meant that it could be flagged to hide it as a default.

Reply:
This has been discussed last week. This week @jhirsi is on vacation but we've decided with @tokangas that we won't flag command usage with any flags. This can be changed later if w come to a different conclusion.

The main reason for not flagging these new arguments is that we haven't done it in the past. `--cellular_timeout` and `--cellular_service` are there always irrespective of `CONFIG_LOCATION_METHOD_CELLULAR` value. Same for Wi-Fi arguments and method type as well as `gnss` command's subcommand `pgps`.
Another point is that do we want to start flagging usages so that user wouldn't have a way to see all arguments at one glance without looking at the code?

So, we decided that at this point we rather stay consistent with what we have.